unlock-js/src/web3Service.js

```diff
@@ -120,10 +121,14 @@ export default class Web3Service extends UnlockService {
    * Temporary function that allows us to use ethers functionality
    * without interfering with web3. This will be moved to the constructor when
    * we remove web3
+   * TODO: ^ assess this?
    */
-  setup(readOnlyProvider) {
+  setup(readOnlyProvider, network = 1) {
```

Comment:
I'm curious if this default is the correct choice. I think it might be better to leave it entirely undefined, and have the fallback behavior be for it to get the network ID on its own, like it does now.

Reply:
That's a very good point. I'll leave as undefined.